Lib/python/pyrun.swg

```diff
@@ -335,6 +335,10 @@ swig_varlink_setattr(PyObject *o, char *n, PyObject *p) {
   return res;
 }
 
+#if defined(Py_LIMITED_API)
+# define SWIG_HEAPTYPES
+#endif
+
```

Comment:
This seems the wrong fix - with this change we'd have multiple different places where this gets defined.

Travelling currently so not easy for me to look but why isn't it getting defined by the code I moved in the commit you referenced?

Reply:
because the code pyruntime.swg is not used (only pyrun.swg has effect here):  in the client code I only include the swig runtime header: here is the same sample code that I used to report a previous error:
https://github.com/jschueller/swig-cmake-example/blob/master/PythonFieldFunction.cxx#L2